src/lib/save_result.ml

```diff
@@ -63,7 +63,7 @@ let make_saving_node
       )
   in
   let name =
-    sprintf "Salvation: %s" (Filename.basename path) in
+    sprintf "Saved: %s" key in
```

Comment:
Could this say something like copied and saved or 'backing-up'? At first, I didn't really get what this was doing as I thought it was the step that actually creates the file.